src/user-guide/sharing-workflows/sharing-workflows.rst

```diff
@@ -0,0 +1,285 @@
+       "group:groupA": ["read", "play", "stop"],
+       "user2": ["!stop"]
+   }
+
+Again, providing your site configuration permits you to grant this access,
+this configuration, for `user2` (who is a member of system group `groupA`) would
+result in them having "read" and "play" access. The the access to stop your workflows
+has been removed and this action will be forbidden.
+
+Access Groups
+^^^^^^^^^^^^^
+For convenience, cylc operations available in the ui have been bundled into
+access groups. These should be capitalized to distinguish from cylc operations.
+
+We currently support ``READ``, ``CONTROL`` and ``ALL`` and to remove permissions
+to operations in these groups, use ``!READ``, ``!CONTROL``, ``!ALL``.
+
+
+.. csv-table:: Access Group Mappings
+   :header: "Operation", "READ", "CONTROL", "ALL"
+
+   "Broadcast", , , "X"
+   "Ext-trigger",, "X", "X"
+   "Hold",, "X", "X"
+   "Kill",, "X", "X"
+   "Message",, "X", "X"
+   "Pause",, "X", "X"
+   "Ping","X", , "X"
+   "Play",, "X", "X"
+   "Poll",, "X", "X"
+   "Read","X", , "X"
+   "Release",, "X", "X"
+   "ReleaseHoldPoint",, "X", "X"
+   "Reload",, "X", "X"
+   "Remove",, "X", "X"
+   "Resume",, "X", "X"
+   "SetGraphWindowExtent",, "X", "X"
+   "SetHoldPoint",, "X", "X"
+   "SetOutputs",, "X", "X"
+   "SetVerbosity",, "X", "X"
+   "Stop",, "X", "X"
+   "Trigger",, "X", "X"
+
+
+.. note::
+
+   ``READ`` access group (which exands to ``["read", "ping"]``) is a method for
+   granting non-interactive access.
+
+.. _user_configuration:
+
+User Authorization Configuration
+--------------------------------
+`c.CylcUIServer.user-authorization`, which is loaded from
+`~/.cylc/hub/jupyter_config.py`, contains your preferences for granting access
+to other users. This configuration should be entered as a Python
+dictionary. If a user does not appear in your user config, the default site
+access will apply.
+You are only permitted to grant access, within the bounds set at site level.
+
+Example User Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An example user configuration:
+
+.. code-block:: python
+
+   c.CylcUIServer.user_authorization = {
+       "*": ["READ"],   
+       "group:groupA": ["CONTROL"],
+       "user1": ["read", "pause", "!play"],
+       "user2": ["!ALL"]
+       }
+
+In this scenario:
+
+- ``"*"``  represents any authenticated user. They have permission to view all
+  workflows and ping them on the gui.
+
+- ``"group:groupA"`` applies ``CONTROL`` permissions to any member of system
+  `groupA`.
+  Note that, since permissions are additive, these users will gain ``READ`` access
+  from the ``"*":["READ"]`` assignment.
+
+- ``"user1"`` will have permission to view workflows, ``pause`` but not ``play``
+  workflows, even if `user1` is a member of the system `groupA`. This is due to
+  negations taking precedence over additions.
+
+- ``"user2"`` is not permitted to view workflows, or perform any operations. 
+
+.. _site_configuration:
+
+Site Authorization Configuration
```

Comment:
We need to add a note somewhere to explain the limitations of this security model to make it clear that perceived weaknesses in the site-config are not shortcomings in Cylc.

Something along the lines of:

> The `site_authorization` configuration allows sites to configure sensible defaults and limits for the permissions users can delegate.
>
> Note, however, that as the UIS runs as the user they have full control over this process so, in theory, may bypass these restrictions in a variety of ways. As an extreme example a user can pass their credentials on to another person. Such behaviours cannot be controlled through technical measures so must be enforced by site policy.